packages/forms/src/UIForm/fields/TimezoneList/TimezoneList.utils.js

```diff
@@ -24,7 +24,7 @@ export function getTimezones(lang, cldrTimezones) {
 	const getTimezoneInfo = (timezone, translatedName) => {
 		const timezoneName = translatedName || get(zones, `${timezone.replaceAll('/', '.')}.exemplarCity`, timezone);
 		const offset = talendUtils.date.getUTCOffset(timezone);
-		const name = `(${talendUtils.date.formatReadableUTCOffset(offset)}) ${timezoneName}`;
+		const name = `(UTC ${talendUtils.date.formatReadableUTCOffset(offset)}) ${timezoneName}`;
```

Comment:
Maybe it's time to cover this with a unit test, WDYT about this?

Reply:
Oh you're right, didn't noticed there wasn't any

Reply:
shouldn't it be internationalised ?

Reply:
We previously had it this way in Portal, so I assumed not 🤔

Reply:
it could be interesting to ask doc and i18n teams, don't you think? 
![image](https://user-images.githubusercontent.com/26482371/123099533-8fbe5180-d432-11eb-940a-0f375897403f.png)

Reply:
UTC is an international acronym isn't it? or at least, very well known

Reply:
Waiting for i18n team insights